cluster-autoscaler/Makefile

```diff
@@ -104,4 +106,19 @@ container-arch-%: build-in-docker-arch-% make-image-arch-%
 test-in-docker: clean docker-builder
 	docker run ${RM_FLAG} -v `pwd`:/cluster-autoscaler/:Z autoscaling-builder:latest bash -c 'cd /cluster-autoscaler && go test -race ./... ${TAGS_FLAG}'
 
+juju-cli-images: $(addprefix juju-cli-image-arch-,$(ALL_ARCH))
+	@echo "Juju CLI images built"
+
+# To use buildx to build multi-arch images, install qemu
+# sudo apt-get install -y qemu-user-static
+juju-cli-image-arch-%:
+	docker buildx build \
```

Comment:
does this say `buildx`? I had to go [look this up](https://docs.docker.com/buildx/working-with-buildx/).

Reply:
there will be images for each arch. so:
```
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-amd64
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-arm64
```

Reply:
and yes buildx is whats used to create multi-arch images from the single juju CLI dockerfile. This works in part because the ubuntu-20.04 image is also multiarch.

Reply:
what kind of TAG numbers? 
```
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-amd64:0
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler-juju-amd64:1
```
or
```
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler:juju-amd64-0
upload.rocks.canonical.com:5000/cdk/cluster-autoscaler:juju-amd64-1
```